apps/frontend/components/editor/EngagementSidebar/index.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <VNavigationDrawer
+    width="380"
+    class="sidebar"
+    color="primary-darken-2"
+    elevation="5"
+    location="right"
+    permanent
+  >
+    <div class="sidebar-container pa-4">
+      <div class="text-body-2 font-weight-bold text-primary-lighten-4 my-4">
+        High Engagement at Scale
+      </div>
+      <VList>
+        <VListItem
+          v-for="(it) in guidelines"
+          :key="it.id"
+          :subtitle="it.description"
+          :title="it.title"
+          class="pa-4"
+          variant="tonal"
+          rounded
+        >
+          <template #prepend>
+            <VBadge
+              color="lime-accent-4"
+              icon="mdi-check-bold"
+              :model-value="progress[it.id]"
+            >
+              <VAvatar variant="tonal">
+                <VIcon>{{ it.icon }}</VIcon>
+              </VAvatar>
+            </VBadge>
+          </template>
+          <VDivider class="my-3" />
+          <div class="d-flex flex-wrap ga-1">
+            <template v-for="(metric, key) in it.metric" :key="key">
+              <VChip v-if="metric" size="small" rounded="pill" pill>
+                <VAvatar
+                  class="font-weight-bold"
+                  :color="progress[it.id] ? 'lime-accent-4' : 'white'"
+                  variant="tonal"
+                  start
+                >
+                  +{{ metric }}
+                </VAvatar>
+                {{ startCase(key) }}
+              </VChip>
+            </template>
+          </div>
+        </VListItem>
+      </VList>
+      <RadarChart :data="chartData" :max="4" :min="0" dark />
+    </div>
+  </VNavigationDrawer>
+</template>
+
+<script lang="ts" setup>
+import sumBy from 'lodash/sumBy';
+import startCase from 'lodash/startCase';
+import { RadarChart } from '@tailor-cms/core-components';
+
+const editorStore = useEditorStore();
+const contentElementStore = useContentElementStore();
+const repositoryStore = useCurrentRepository();
+
+const { $ceRegistry, $schemaService } = useNuxtApp() as any;
+
+const guidelines = computed(() => {
+  if (!repositoryStore.selectedActivity) return [];
+  const { type } = repositoryStore.selectedActivity;
+  return $schemaService.getLevel(type)?.guidelines(
```

Comment:
See store comment, consider replacing with selectedActivityGuidelines there and reuse here.